src/test/java/uk/gov/hmcts/dts/fact/model/AdditionalLinkTest.java

```diff
@@ -26,10 +27,40 @@ void testCreation(boolean welsh) {
             Locale locale = new Locale("cy");
             LocaleContextHolder.setLocale(locale);
         }
+        entity.setLocation(new SidebarLocation(1, "Find out more about"));
         AdditionalLink additionalLink = new AdditionalLink(entity);
-        assertThat(additionalLink.getUrl()).isEqualTo(entity.getUrl());
-        assertThat(additionalLink.getDescription()).isEqualTo(welsh ? entity.getDescriptionCy() : entity.getDescription());
+
+        final SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(additionalLink.getUrl()).isEqualTo(entity.getUrl());
+        softly.assertThat(additionalLink.getDescription()).isEqualTo(welsh ? entity.getDescriptionCy() : entity.getDescription());
+        softly.assertThat(additionalLink.getLocation()).isEqualTo(entity.getLocation().getName());
+        softly.assertAll();
 
         LocaleContextHolder.resetLocaleContext();
     }
+
+    @Test
+    void testCreationWhenSidebarLocationIsNull() {
+        entity.setLocation(null);
```

Comment:
Isn't location not null in the migration?

Reply:
I know....it is to get over the sonarqube complaint on low code coverage in tests. I think I can remove the null check from the model class and remove this test here